samples/AzureFunctionsApp/Entities/Lifetime.cs

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Net;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.DurableTask.Client;
+using Microsoft.DurableTask.Client.Entities;
+using Microsoft.DurableTask.Entities;
+using Microsoft.Extensions.Logging;
+
+namespace AzureFunctionsApp.Entities;
+
+/// <summary>
+/// Example showing the lifetime of an entity. An entity is initialized on the first operation it receives and then
+/// is considered deleted when <see cref="TaskEntity{TState}.State"/> is <c>null</c> at the end of an operation.
+/// </summary>
+public class Lifetime : TaskEntity<MyState>
+{
+    readonly ILogger logger;
+
+    public Lifetime(ILogger<Lifetime> logger)
+    {
+        this.logger = logger;
+    }
+
+    /// <summary>
+    /// Optional property to override. When 'true', this will allow dispatching of operations to the TState object if
```

Comment:
What does "dispatching of operations to the TState object" mean? I don't really understand what this is describing.